refinery/core/models.py

```diff
@@ -1346,43 +1346,6 @@ def send_email(self):
                 "'%s' with UUID '%s'",
                 self.get_status(), user.email, name, self.uuid)
 
-    def rename_results(self):
-        """Rename files in file_store after download"""
-        logger.debug("Renaming analysis results")
-        # rename file_store items to new name updated from galaxy file_ids
-        for result in self.results.all():
-            try:
-                item = FileStoreItem.objects.get(uuid=result.file_store_uuid)
-            except (FileStoreItem.DoesNotExist,
-                    FileStoreItem.MultipleObjectsReturned) as exc:
-                logger.error("Error renaming analysis result '%s': %s",
-                             result, exc)
-                break
-
-            # workaround for FastQC reports downloaded from Galaxy as zip
-            # archives
-            (root, ext) = os.path.splitext(result.file_name)
-            if ext == '.html':
-                try:
-                    zipfile = FileType.objects.get(name='ZIP')
-                except (FileType.DoesNotExist,
-                        FileType.MultipleObjectsReturned) as exc:
-                    logger.error("Error renaming HTML to zip: %s", exc)
-                else:
-                    if item.filetype == zipfile:
-                        item.rename_datafile(''.join([root, '.zip']))
-            else:
-                item.rename_datafile(result.file_name)
```

Comment:
It looks like you've lost this functionality. Have you tested this branch with any workflows?

Reply:
i don't think the functionality is not lost - this appears to be redundant data processing given that something identical happens here: https://github.com/refinery-platform/refinery-platform/blob/develop/refinery/analysis_manager/tasks.py#L498.  the only difference is that this pull request now sets the name in flight: https://github.com/refinery-platform/refinery-platform/pull/3471/files/6e8fc3b2b1047e3429121406983fe438f88cce8a#diff-49577ffb85a5a9e95447df9c8ccc34eeR523

Reply:
yes, i can do that

Reply:
Also, can `FileStoreItem.rename_datafile()` be removed now?

Reply:
We can remove it, yes